python/ray/autoscaler/aws/example-full.yaml

```diff
@@ -74,7 +74,7 @@ available_node_types:
             # Default AMI for us-west-2.
             # Check https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/_private/aws/config.py
             # for default images for other zones.
-            ImageId: ami-0387d929287ab193e
```

Comment:
Could you please also update the other yaml files in this directory if needed?

Reply:
I believe it's not needed. The only two other examples using hardcoded image identifiers are just for Java, using a plain Ubuntu 18.04 image, and for internal development, using a plain Ubuntu 16.04 image. None of those are related to this pull request, covering only the DLAMI identifiers.

https://github.com/ray-project/ray/blob/3f4c4247505f9f1018c73ac3cdb3bfd1f60c6bf2/python/ray/autoscaler/aws/example-java.yaml#L30

https://github.com/ray-project/ray/blob/3f4c4247505f9f1018c73ac3cdb3bfd1f60c6bf2/python/ray/autoscaler/aws/development-example.yaml#L34